### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_ScreenReader: Voiceover does not announce the name of the field.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9319]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Launch Voiceover and navigate to the 4 edit fields under storage settings button.

**Actual:**
On navigating to the 4 edit fields under the storage settings button using tab key the value in the field and the control of the field is announced.

**Expected:**
The name of the field control and value should be announced.

**User Impact:**
Screen readers users will not know what data needs to be entered.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/Edwfjm9x9UlKns6KcqKGQvUB_6v3yBPGhN5taYyyNlpETQ?e=nZAyep

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/azuredatastudio/issues/9345

**Comment by sspatil123** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
